Sloth.Web/Controllers/TeamsController.cs

```diff
@@ -143,17 +143,38 @@ public async Task<IActionResult> CreateUserRole(string teamId, string userId, st
         {
             // fetch team from db
             var team = await DbContext.Teams
-                .Include(t => t.ApiKeys)
+                .Include(a => a.UserTeamRoles).ThenInclude(r => r.Role)
```

Comment:
this is going to be pretty inefficient because it has to query all users in a team and bring them back, and team info for each one.  Better to just get the team, and then do another query if you want to see the other roles that user has.